siriuspy/siriuspy/clientconfigdb/configdb_client.py

```diff
@@ -170,6 +171,39 @@ def check_valid_value(self, value, config_type=None):
         config_type = self._process_config_type(config_type)
         return _templates.check_value(config_type, value)
 
+    @staticmethod
+    def compare_configs(config1, config2, pos_pattern=None, neg_pattern=None):
+        """Compare "pvs" attribute of configs 1 and 2.
+
+        Print the differences.
+
+        Args:
+            config1 (dict): Valid configuration with "pvs" attribute.
+            config2 (dict): Valid configuration with "pvs" attribute.
+            pos_pattern (str, optional): Only PVs with this pattern will be
+                checked. Defaults to None. None means all PVs will be compared.
+            neg_pattern (_type_, optional): PVs which match this pattern will
+                not be checked. Defaults to None. None means no PV will be
+                excluded from comparison. In case of conflict with
+                `pos_pattern`, it will take precedence.
+        """
+        pos_re = _re.compile('' if pos_pattern is None else pos_pattern)
+        # In case of None in neg_pattern, create a regexp that never matches:
+        # https://stackoverflow.com/questions/1723182/a-regex-that-will-never-be-matched-by-anything
+        neg_re = _re.compile(r'(?!x)x' if neg_pattern is None else neg_pattern)
+
+        cf1pvs = {n: v for n, v, t in config1['pvs']}
+        cf2pvs = {n: v for n, v, t in config2['pvs']}
```

Comment:
couldn't be `{n: v for n, v, _ in config1['pvs']}` since `t` is not being used?

Reply:
right, done!